docs/reference/sql/endpoints/rest.asciidoc

```diff
@@ -235,6 +235,81 @@ Douglas Adams  |The Hitchhiker's Guide to the Galaxy|180            |1979-10-12T
 // TESTRESPONSE[s/\|/\\|/ s/\+/\\+/]
 // TESTRESPONSE[_cat]
 
+[[sql-rest-columnar]]
+[float]
+=== Columnar results
+
+The most well known way of displaying the results of an SQL query result in general is the one where each
+individual record/document represents one line/row. For certain formats, {es-sql} can return the results
+in a columnar fashion: one row represents all the values of a certain column from the current page of results.
+
+A columnar style of results makes sense for certain formats, thus it's available for `json`, `yaml`, `cbor` and `smile`.
```

Comment:
To avoid the "for certain formats" repetition, this could be rephrased to "The following formats can be returned in columnar orientation".